config/standard/nools-extras.js

```diff
@@ -230,6 +230,20 @@ function countReportsSubmittedInWindow(reports, form, start, end) {
   return reportsFound;
 }
 
+function countANCVisits(reports, start, end){
+  var reportsFound = 0;
+  reports.forEach(function(r) {
+    if (antenatalForms.indexOf(r.form) >= 0) {
```

Comment:
I see that this function is mostly copied from an existing one, but figured I'd point out that since this config is now running on 3.x, with the newer browser requirements you can now use `includes()`.